lightly/models/modules/heads.py

```diff
@@ -128,21 +128,62 @@ class MoCoProjectionHead(ProjectionHead):
     """Projection head used for MoCo.
 
     "(...) we replace the fc head in MoCo with a 2-layer MLP head (hidden layer
-    2048-d, with ReLU)" [0]
+    2048-d, with ReLU)" [1]
 
-    [0]: MoCo, 2020, https://arxiv.org/abs/1911.05722
+    "The projection head is a 3-layer MLP. The prediction head is a 2-layer MLP. The
+    hidden layers of both MLPs are 4096-d and are with ReLU; the output layers of both
+    MLPs are 256-d, without ReLU. In MoCo v3, all layers in both MLPs have BN" [2]
 
+    [0]: MoCo v1, 2020, https://arxiv.org/abs/1911.05722
+    [1]: MoCo v2, 2020, https://arxiv.org/abs/2003.04297
+    [2]: MoCo v3, 2021, https://arxiv.org/abs/2104.02057
     """
 
     def __init__(
-        self, input_dim: int = 2048, hidden_dim: int = 2048, output_dim: int = 128
+        self,
+        input_dim: int = 2048,
+        hidden_dim: int = 4096,
+        output_dim: int = 256,
+        num_layers: int = 2,
```

Comment:
This should be 3 for v3 I believe.

Reply:
I just discussed this with @guarin and changing the defaults will change user code so we'd prefer to use the v2 settings as default values for now.